src/app/patient-table/patient-table.component.ts

```diff
@@ -46,21 +54,21 @@ export class PatientTableComponent implements OnInit, OnChanges {
     ];
 
     ngOnChanges(changes: SimpleChanges): void {
```

Comment:
`ngOnChanges`  - [read](https://medium.com/@isaacplmann/ngonchanges-only-runs-when-the-input-change-comes-from-a-template-binding-like-component-8797b759ba0b#:~:text=Share-,ngOnChanges%20only%20runs%20when%20the%20Input%20change%20comes%20from%20a,that%20you've%20changed%20something.)

This will not work, the logic have to go to a function that you want to trigger when you have a results coming back from modal